src/components/table/JobTable.tsx

```diff
@@ -1,173 +1,177 @@
-    };
-
-    const handleSeeMoreOpen = (jobId: string) => {
-      const params = _.clone(query || {});
-      params.expandedJob = jobId;
-      push(`${location.pathname}?${urlEncodeParams(params)}`);
-    };
+function JobTable({ jobs, handleFilterJobs, loading, searchString, query, refreshCnt }: IProps) {
+  const handleSeeMoreOpen = (jobId: string) => {
+    const params = _.clone(query || {});
+    params.expandedJob = jobId;
+    push(`${location.pathname}?${urlEncodeParams(params)}`);
+  };
+  const theme = useTheme() as CustomTheme;
 
-    const columns: Array<TableColumnType> = [
-      {
-        label: 'Job ID',
-        name: 'id',
-        allowSort: false,
-        width: '20%',
-        render: (row: any) => (
-          <div>
-            <button
-              type="button"
-              onClick={() => handleSeeMoreOpen(row.id)}
-              className={classes.seeMoreLink}
-            >
-              <span>{`${row.id || 'See More'}`}</span>
-            </button>
-            <CopyTextButton valueToCopy={row.id} nameOfValue="Job ID" />
-          </div>
-        ),
-      },
-      {
-        label: 'Class',
-        name: 'class_name',
-        allowSort: false,
-        width: '15%',
-        render: (row: any) => (
-          <span title={row.class_name}>{_.last(row.class_name?.split('.'))}</span>
-        ),
-      },
-      {
-        label: 'Description',
-        name: 'description',
-        allowSort: false,
-        width: '45%',
-      },
-      {
-        label: 'Date',
-        name: 'submitted',
-        allowSort: true,
-        render: (row: any) => (
-          <span title={row?.submitted}>{moment(row?.submitted).fromNow()}</span>
-        ),
-        width: '10%',
-      },
-      {
-        label: 'Status',
-        name: 'job_status',
-        allowSort: false,
-        width: '10%',
-        render: (row: any) => (
-          <span className={classes.statusContainer} title={statusMap[row.job_status].label}>
-            <i className={clsx(classes.statusIcon, statusMap[row.job_status].icon)} />
-            <span>{statusMap[row.job_status].label}</span>
-          </span>
-        ),
-      },
-      {
-        label: 'Duration',
-        name: 'job_duration',
-        allowSort: false,
-        width: '10%',
-        render: (row: any) => {
-          if (row.submitted && row.completed) {
-            const duration = moment(row.completed).diff(moment(row.submitted), 'seconds');
-            const durationStr = moment
-              .duration(duration, 'seconds')
-              .format('h [hrs] m [min] s [sec]');
-            return <span title={durationStr}>{durationStr}</span>;
-          }
-          return '--';
-        },
+  const columns: Array<TableColumnType> = [
+    {
+      label: 'Job ID',
+      name: 'id',
+      allowSort: false,
+      width: '20%',
+      render: (row: JobModel) => (
+        <div>
+          <SeeMoreLink onClick={() => handleSeeMoreOpen(row.id)} theme={theme}>
```

Comment:
This seems like an anti-pattern to me: I don't think we should be passing around the theme.